src/main/java/apoc/util/Util.java

```diff
@@ -686,4 +686,10 @@ public static boolean classExists(String className) {
             return null;
         }
     }
+
+    public static Optional<String> getLoadUrlByConfigFile(String loadType, String key, String urlOrHost){
+        key = key + (key == null || key.isEmpty() ? "" : "." + urlOrHost);
```

Comment:
```suggestion
        key = Optional.ofNullable(key).map(s -> s + "." + urlOrHost).orElese(StringUtils.EMPTY);
```
In futuro i null gestiamoli sempre con gli Optional